packages/clerk-js/src/ui/hooks/useDelayUnmount.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+
+export function useDelayUnmount<T>(isMounted: T, delayTime: number) {
```

Comment:
❓ Can you please elaborate on this hook?

Reply:
So this allow me to delay the unMount of a component in order to allow the "exit" animation to run before the actual unmount happens and the node is removed from the dom

Reply:
Thanks for the extra information. Consider naming the hook to `useDelayedUnmount`